register.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>signup</title>
+    <link rel="stylesheet" href="./CSS/register.css">
+    <link rel="stylesheet" href="./CSS/palette.css">
+</head>
+<body>
+    <main class="main">
+        <section class="main-box">
+            <section class="main-box-logo">
+                <a href="./index.html">
+                    <img class="login-logo" src="./image/Property 1=lg.png" alt="판다마켓">
+                </a>
+            </section>
+            <section class="sub-box">
+                <p class="input-name">이메일</p>
+                <input class="input-tag email" type="email" placeholder="이메일을 입력해주세요">
+                <div class="error email-error"></div>
+                <p class="input-name">닉네임</p>
+                <input class="input-tag" type="username" placeholder="닉네임을 입력해주세요">
+                <div class="error username-error"></div>
+                <p class="input-name input-pass">비밀번호</p>
+                <div class="input-pass-box">
+                    <input class="input-tag" type="password" placeholder="비밀호를 입력해주세요">
+                    <img class="password-icon" src="./image/gnb/btn_visibility_on_24px.png" alt="비밀번호 안보이게">
+                </div>
+                <div class="error password-error"></div>
+                <p class="input-name input-pass">비밀번호 확인</p>
+                <div class="input-pass-box">
+                    <input class="input-tag" type="passwordConfirm" placeholder="비밀호를 다시 한 번 입력해주세요">
+                    <img class="password-icon" src="./image/gnb/btn_visibility_on_24px.png" alt="비밀번호 안보이게">
+                </div>
+                <div class="error passwordConfirm-error"></div>
```

Comment:
존재하지 않는 naming convention이네요!
`camelCase`와 `kebab-case`가 섞인 모습입니다.

```suggestion
                <div class="error password-confirm-error"></div>
```